msobooking.yaml

```diff
@@ -0,0 +1,436 @@
+      publicsafety.employeeid:
+        type: "publicsafety.employeeid"
+        column: "DNA_SAMPLE_OFFICER"
+    name: "dnaofficer"
+
+  committingofficer:
+    fqn: "ol.employee"
+    entitySetName: "MSOOfficer"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "COMOFF"
+      publicsafety.employeeid:
+        type: "publicsafety.employeeid"
+        column: "COMOFF"
+    name: "committingofficer"
+
+  committingcounty:
+    fqn: "ol.location"
+    entitySetName: "MSOCommittingcounty"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "COUNTY"
+      ol.county:
+        type: "ol.county"
+        column: "COUNTY"
+    name: "committingcounty"
+
+  releaselocation:
+    fqn: "ol.location"
+    entitySetName: "MSOReleaselocation"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "RELLOC"
+      location.name:
+        type: "location.name"
+        column: "RELLOC"
+      general.notes:
+        type: "general.notes"
+        column: "RELADDR1"
+    name: "releaselocation"
+
+  immigrationstatus:
+    fqn: "ol.immigrationstatus"
+    entitySetName: "MSOImmigrationstatus"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "INS"
+      ol.status:
+        type: "ol.status"
+        column: "INS"
+    name: "immigrationstatus"
+
+  timeserved:
+    fqn: "ol.jailstaylength"
+    entitySetName: "MSOTimeserved"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "SERVED"
+      criminaljustice.timeserveddays:
+        type: "criminaljustice.timeserveddays"
+        column: "SERVED"
+    name: "timeserved"
+
+  persondetails:
+    fqn: "ol.persondetails"
+    entitySetName: "MSOPersondetails"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "PCP"
+      person.maritalstatus:
+        type: "person.maritalstatus"
+        column: "MARITAL"
+      person.ageatevent:
+        type: "person.ageatevent"
+        column: "AGE"
+    name: "persondetails"
+
+  persondetailscriminaljustice:
+    fqn: "ol.persondetailscriminaljustice"
+    entitySetName: "MSOPersondetailscriminaljustice"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "SENTENCED_AS_ADULT"
+      criminaljustice.treatasadult:
+        type: "criminaljustice.treatasadult"
+        column: "SENTENCED_AS_ADULT"
+    name: "persondetailscriminaljustice"
+
+associationDefinitions:
+  has:
+    fqn: "ol.has"
+    entitySetName: "MSOHas"
+    src: "person"
+    dst: "persondetails"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "assn_has"
+    name: "has"
+
+  has2:
+    fqn: "ol.has"
+    entitySetName: "MSOHas"
+    src: "person"
+    dst: "persondetailscriminaljustice"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "assn_has2"
+    name: "has2"
+
+  sentencedwith:
+    fqn: "ol.sentencedwith"
```

Comment:
Either here, or in sentence, we should add a date. If there is no explicit sentence date, use the commit date AND be sure to write that into the description of the datetime property so users know that's what we did.

Reply:
Added COMDATE as the property datetime.received in sentence.